lib/rules/link-rel-noopener.js

```diff
@@ -88,6 +97,41 @@ module.exports = class LinkRelNoopener extends Rule {
   }
 };
 
+function isFixable(elementNode) {
+  let oldRel = AstNodeInfo.findAttribute(elementNode, 'rel');
+  return !oldRel || AstNodeInfo.isTextNode(oldRel.value);
+}
+
+function fix(node, config) {
+  let oldRel = AstNodeInfo.findAttribute(node, 'rel');
+
+  if (!isFixable(node)) {
+    return;
+  }
```

Comment:
Since the `fix` function is only invoked from within a `isFixable` guard we can probably remove this one as a bit redundant?

Reply:
Ah yep. I was trying to be clever and have `if (this.mode === fix && fix(node, config)) { return } else { this.log(...) }` but prettier/eslint didn't like that style so I backed out of it. This is removed and updated. Thanks for the review!